content/blog/2020/hello-world.md

```diff
@@ -12,7 +13,7 @@ draft: false
 ## Introduction
 
 This article should serve as a "getting started" guide for developers who are considering integrating BDK in their projects: it tries to introduce the reader to the basic concepts behind the library and some of its
-modules and components that can be used to build a very simple functioning Bitcoin wallet. All the information written in this article are valid for the current `master` git branch, and should remain valid for the upcoming [`v0.2.0` release](https://github.com/bitcoindevkit/bdk/projects/1)
+modules and components that can be used to build a very simple functioning Bitcoin wallet. All the information written in this article are valid for the current `master` git branch, and should remain valid for the upcoming [`v0.11.0` release](https://github.com/bitcoindevkit/bdk/projects/1)
```

Comment:
The `0.11.0` release is out now, and it's better to link to the cargo version since that's where folks will find the latest api docs. 

```suggestion
modules and components that can be used to build a very simple functioning Bitcoin wallet. All the information written in this article are valid for the current `master` git branch, and should remain valid for the [`v0.11.0` release](https://crates.io/crates/bdk/0.11.0)
```